Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -711,17 +779,139 @@
             return regexp.test(Url.toLowerCase());
         };
 
-        // show usageDetailsGrid - takes a itSystemUsageId for data and systemName for modal title
-        public showUsageDetails = (usageId, systemName) => {
-            //Filter by usageId
-            this.usageGrid.dataSource.filter({ field: "ItSystemId", operator: "eq", value: usageId });
+        // show usageDetailsGrid - takes a itSystemId for data and systemName for modal title
+        public showUsageDetails = (systemId, systemName) => {
+            this.resetMigrationFlow(); //Migration flow starts freshly from this function
+            this.oldItSystemName = systemName;
+            //Filter by systemId
```

Comment:
kommentaren er lidt fjollet, for den siger ikke noget om hvorfor. Måske linjen skulle trækkes ud i en funktion så som. `filerUsageGridByCurrentSystem()` eller sådan noget

Reply:
Denne linje har @JacobMalling rettet til og det er ikke længere et problem for serveren serverer kun det rigtige data i det nye kald